app/views/shared/_add_repos_list.html.slim

```diff
@@ -1,5 +1,7 @@
-- if repos.blank?
-  p None!
+- if repos.blank? || repos.count == 0
+  table.table.table-striped
+    tr
```

Comment:
Hi @prathamesh-sonpatki ,

To be honest, I was a bit confused too. I just figured out that the repos variable in the code above is not an array but an instance of the SortedRepoCollection class. That's why even though repos doesn't have a content, 'repos.blank?' returns false.
For instance:

![screen shot 2015-06-30 at 1 18 26 pm](https://cloud.githubusercontent.com/assets/6015897/8424284/d728f14c-1f2a-11e5-8aa6-c6e6f198f02c.png)

Where @starred_repos will be passed:
![screen shot 2015-06-30 at 1 18 55 pm](https://cloud.githubusercontent.com/assets/6015897/8424290/e2241036-1f2a-11e5-95ba-79f8dc3b95e7.png)

ReposController
![screen shot 2015-06-30 at 1 19 23 pm](https://cloud.githubusercontent.com/assets/6015897/8424293/e96a8dc0-1f2a-11e5-9d33-cc369ea74ebd.png)